### PR TITLE
Add make proxivocab optional

### LINKED ISSUE #224: Make it optional if proxivocab details should be included in exports
image

1.  Add a new download icon right to the existing one, which downloads the vocab with proxivocab information (please use a second icon for that OR create a popup with two buttons "Normal Download" and "Proxivocab Enhanced")
2. Update the behavior of the existing ("normal") download button so that it download the vocab without proxivocab information

image

### PR DESCRIPTION
**Pull request by alihussainiam**
closes #224

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,55 +1,5 @@
-# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
```
Comment (JohannesLipp) [inline .gitignore:1]: I suggest to keep these entries in the gitignore for people that use a JetBrain IDE for contributing to this project.

File: app/src/app/home-dashboard/vocabulary-list/vocabulary-list.component.ts [hunk 2]
```diff
@@ -15,79 +15,134 @@ import { N3Codec } from '../../../app/mxgraph/N3Codec';
...
+            nzContent: '<b>You will lose all access to this vocabulary. This action cannot be undone.</b>',
+            nzOnOk: () => this.vocabService.deleteVocabulary(vocab_id)
+        });
+    }
 
-  downloadVocab(id: string, name: string) {
-      console.log('downloading vocabulary: ' + id + '...');
+    /**
+     *
+     * @param vocab_id TODO (Johannes): TBD
```
Comment (JohannesLipp) [inline app/src/app/home-dashboard/vocabulary-list/vocabulary-list.component.ts:40]: Please open an issue so that we do not forget about this, thanks!

**Review by JohannesLipp — CHANGES_REQUESTED** [review]
Good work, thank you! Please just revert the deletions from the gitignore, so we can merge it, thanks!

File: .gitignore (hunk 1 above)
Comment (alihussainiam) [reply .gitignore:1]: [a476fe0](https://github.com/Semantic-Society/Neologism/pull/240/commits/a476fe0cb733f12df92b14a8222af24c3388e674) solves this issue. thank you

**Pull request merged**